pkgs/applications/version-management/git-cola/default.nix

```
@@ -1,21 +1,31 @@
-{ stdenv, lib, fetchFromGitHub, python3Packages, gettext, git, qt5, gitUpdater }:
+{ stdenv
+, lib
+, fetchFromGitHub
+, python3Packages
+, gettext
+, git
+, qt5
+, gitUpdater
+}:
 
 python3Packages.buildPythonApplication rec {
   pname = "git-cola";
-  version = "4.5.0";
+  version = "4.6.1";
```

```suggestion
  version = "4.6.1";  
  pyproject = true;

```
This usually gets placed near the beginning